cdap-ui/app/directives/dag/my-dag-ctrl.js

```diff
@@ -247,6 +247,7 @@ angular.module(PKG.name + '.commons')
           e.el.style.left = '0px';
           e.el.style.top = '0px';
           transformCanvas(e.pos[1], e.pos[0]);
+          NodesActionsFactory.resetPluginCount(vm.panning);
```

Comment:
This is a little mis-leading. The funtion is about resetting plugin counts but we are passing in the panning for the canvas and resetting the canvas panning. It would probably be better if we split into two functions.